docs/stability.rst

```diff
@@ -252,7 +252,7 @@ The current planned and existing sub-packages are:
                 <span class="stable"></span>
             </td>
             <td>
-                New in v0.2. Adapted from <tt class="docutils literal"><span class="pre">pnbody</span></tt> and integrated into Astropy. Current functionality stable with intent to maintain backwards compatibility. Significant new functionality is likely to be added in future versions.
+                New in v0.2. Current functionality stable with intent to maintain backwards compatibility. Significant new functionality is likely to be added in future versions.
```

Comment:
@astrofrog @pllim - yes, in this case I was referring to support for `Distribution`s or similar.  At some point I think `Quantity` *has* to support uncertainties.  Whether that's technically in `units` or not I guess is a question, but I think its fair to say that, from the user perspective, additions to working with `Quantity` count as `units`.